packages/pangea-node-sdk/src/types.ts

```diff
@@ -660,8 +660,26 @@ export namespace Intel {
       breach_count: number;
     }
 
-    export interface BreachedResult extends Intel.CommonResult {
-      data: BreachedData;
+    export interface BreachedRequest {
+      /** The ID of a breach returned by a provider. */
+      breach_id: string;
+
+      /** Echo the API parameters in the response. */
+      verbose?: boolean;
+
+      /** Get breach data from this provider. */
+      provider?: string;
+    }
+
+    export interface BreachedResult {
+      /** A flag indicating if the lookup was successful. */
+      found: boolean;
+
+      /** Breach details given by the provider. */
+      data?: object;
+
+      /** The parameters, which were passed in the request, echoed back. */
+      parameter?: Dictionary;
```

Comment:
I tried out a request and this came back plural.
```suggestion
      parameters?: Dictionary;
```